keyboards/dztech/dz68rgb/dz68rgb.c

```diff
@@ -0,0 +1,165 @@
+#include "dz68rgb.h"
+#include "config.h"
```

Comment:
This line isn't actually necessary, and should be removed. 
```suggestion
```

Specifically, the build process automatically includes the config.h files, so there is no need to manually add it.